content/posts/see_presentation.Rmd

```diff
@@ -21,36 +21,67 @@ include_graphics("https://github.com/easystats/see/raw/master/man/figures/logo.p
 
 
 
-## see
+## The see package
 
-[We](https://github.com/orgs/easystats/people) have recently decided to collaborate around the new [**easystats project**](https://github.com/easystats), a set of packages designed to make your life *easier*. This project encompasses several packages, devoted for instance to [model access](https://github.com/easystats/insight) or [Bayesian analysis](https://github.com/easystats/bayestestR), [indices of model performance](https://github.com/easystats/performance) or  [visualisation](https://github.com/easystats/see).
+[We](https://github.com/orgs/easystats/people) have recently decided to collaborate around the new [**easystats project**](https://github.com/easystats), a set of packages designed to make your life easier. This project encompasses several packages, devoted for instance to [model access](https://github.com/easystats/insight) or [Bayesian analysis](https://github.com/easystats/bayestestR), [indices of model performance](https://github.com/easystats/performance) or [visualisation](https://github.com/easystats/see).
 
-***Without further ado, please let us introduce the latest addition to the easyverse; the ***  [**`see`**](https://github.com/easystats/see) ***package!***
+***Without further ado, please let us introduce the latest addition to the easyverse; the see ackage!***
 
 
-<!-- The goal of [`performance`](https://github.com/easystats/performance) is to provide lightweight tools to **assess and check the quality of your model**. It includes functions such as [`R2`](https://easystats.github.io/performance/reference/r2.html) for many models (including logistic, mixed and Bayesian models), [`ICC`](https://easystats.github.io/performance/reference/icc.html) or helpers to check [`convergence`](https://easystats.github.io/performance/reference/check_convergence.html), [`overdipsersion`](https://easystats.github.io/performance/reference/check_overdispersion.html) or [`zero-inflation`](https://easystats.github.io/performance/reference/check_zeroinflation.html). See the **list of functions** [here](https://easystats.github.io/performance/reference/index.html). -->
+The goal of [`see`](https://github.com/easystats/see) is to be the visualisation companion to other functions and packages in **easystats**. See the **list of functions** [here](https://easystats.github.io/see/reference/index.html). However, it also includes some nice themes and geoms:
```

Comment:
"...in the easyverse." 

Not sure if we want to start using "easystats" or "easyverse" more...?